packages/kernel/py/stlite-server/stlite_server/server.py

```diff
@@ -152,7 +160,7 @@ def receive_http(
             on_response(404, {}, b"No handler found")
             return
         method_name = method.lower()
-        if method_name not in ("get", "post"):
+        if method_name not in ("get", "post", "delete"):
             on_response(405, {}, b"Now allowed")
```

Comment:
```suggestion
            on_response(405, {}, b"Not allowed")
```

This is a typo I introduced in the past.
Let me suggest the fix here just because I found it during this review.